### PR TITLE
Basic button logic implemented

### PR DESCRIPTION
**Pull request by s-kettle**
Logic has been added for - 

- Add listing button on MainActivity
- Back button on ViewListingActivity
- Cancel button on AddListingActivity

Databinding has been initialised and implemented in the .xml layouts.

### CODE REVIEW

File: app/src/main/java/com/springsamurais/toyswap/ui/addlisting/AddListingClickHandlers.kt [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+import android.content.Context
+import android.content.Intent
+import android.view.View
+import com.springsamurais.toyswap.ui.mainactivity.MainActivity
+
+class AddListingClickHandlers(private var context: Context) {
+
+    fun captureImageButtonClicked(view: View) {
+        // Moazam has code for opening camera
```
Comment (s-kettle) [inline app/src/main/java/com/springsamurais/toyswap/ui/addlisting/AddListingClickHandlers.kt:11]: Before merge, @mmoazam could potentially add camera logic here if we're happy with that.

File: app/src/main/java/com/springsamurais/toyswap/ui/addlisting/AddListingClickHandlers.kt [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    }
+
+    fun accessGalleryButtonClicked(view: View) {
+        // Moazam has code for accessing gallery
```
Comment (s-kettle) [inline app/src/main/java/com/springsamurais/toyswap/ui/addlisting/AddListingClickHandlers.kt:15]: Before merge, @mmoazam could potentially add the open gallery logic if it's ready to go.

**Review by s-kettle — COMMENTED** [review]
I've added some comments for @mmoazam regarding camera logic. I'd like this in before a merge if possible.

**Review by mmoazam — COMMENTED** [review]
I take it the later AddListingActivity.kt will overwrite this one.

**Review by mmoazam — APPROVED** [review]
missed the approve

**Pull request merged**